integration-tests/__tests__/wallet/transfer-ticket-operation.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { CONFIGS } from '../../config';
+import { DefaultContractType, TezosToolkit } from '@taquito/taquito';
+import { ticketsSendTz } from '../../data/code_with_ticket_transfer';
+import { RpcClient, TicketTokenParams } from '@taquito/rpc';
+
+CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
+  const Tezos = lib;
+  const client = new RpcClient(rpc);
+
+  let ticketSendContract: DefaultContractType;
+  let recipient: TezosToolkit;
+  let sender: TezosToolkit;
+  let recipientPkh: string;
+  let senderPkh: string
+  let ticketToken: TicketTokenParams;
+
+  describe(`Transfer tickets between implicit accounts using: ${rpc}`, () => {
+
+    beforeAll(async () => {
+      await setup(true);
+      try {
+        recipient = await createAddress();
+        sender = await createAddress();
+
+        recipientPkh = await recipient.signer.publicKeyHash();
+        senderPkh = await sender.wallet.pkh();
+
+        const fundSender = await Tezos.contract.transfer({ to: senderPkh, amount: 5 });
+        await fundSender.confirmation();
+
+        const ticketSendOrigination = await Tezos.contract.originate({ code: ticketsSendTz, storage: null });
+        await ticketSendOrigination.confirmation();
+
+        ticketSendContract = await ticketSendOrigination.contract();
+        ticketToken = { ticketer: ticketSendContract.address, content_type: { prim: 'string' }, content: { string: 'Ticket' } };
+
+        // Send 3 tickets from the originated contract to sender
+        const sendTickets = await ticketSendContract.methods.default(senderPkh, '3').send();
```

Comment:
Maybe using `methodObjects` can avoid the older syntax?

Reply:
good point, I'll amend